npm-shrinkwrap.json

```diff
@@ -6,7 +6,7 @@
       "version": "1.0.3"
     },
     "abbrev": {
-      "version": "1.0.7"
+      "version": "1.0.9"
```

Comment:
npm 3 will also try to make the tree as flat as possible, and depends on install order for its structure: https://docs.npmjs.com/how-npm-works/npm3-nondet